src/sensors/fake_imu.cpp

```diff
@@ -93,6 +110,16 @@ void FakeImuFromFile::getData(ImuData* imu)
 {
   data::State state = data_.getStateMachineData().current_state;
   bool operational = true;
+  // Random point of failure after acc from 0 to 20 seconds
+  if (state == data::State::kAccelerating && is_fail_acc_) {
+    // Generate a random time for a failure
+    failure_time_acc_ = (rand() % 20 + 1) * 1000000;
```

Comment:
failure_time_acc_ this will change every time getData is called. Only do it once.